app/templates/components/nav-bar.hbs

```diff
@@ -21,7 +21,6 @@
             <a href="{{href-to 'my-sessions'}}" class="item">{{t 'My Sessions'}}</a>
             <a href="{{href-to 'events.list' 'live'}}" class="item">{{t 'Manage Events'}}</a>
             <div class="divider"></div>
-            <a href="{{href-to 'profile'}}" class="item">{{t 'Profile'}}</a>
             <a href="{{href-to 'settings.contact-info'}}" class="item">{{t 'Settings'}}</a>
```

Comment:
@mrsaicharan1 I think you should change the href to `settings.profile` here

Reply:
@shreyanshdwivedi I don't think we need it here because I've removed the link to the profile from the nav-bar. It's a deleted line.

Reply:
@mrsaicharan1 I'm specifying the line number 24 after deletion of the link to the profile -
`<a href="{{href-to 'settings.contact-info'}}" class="item">{{t 'Settings'}}</a>` 
I'm saying the link to `Settings` here must be `settings.profile` because it will redirect to `contact-info` for now. Am I understanding something wrong here?

Reply:
@shreyanshdwivedi Oh okay! So you want me to redirect it to the first tab? Gotcha!